route_by_latency_test.go

```
@@ -0,0 +1,168 @@
+package valkey
+
+import (
+	"fmt"
+	"net"
+	"sync"
+	"testing"
+	"time"
+)
+
+func TestPickLowestLatencyReplica(t *testing.T) {
+	l1, err := net.Listen("tcp", "127.0.0.1:0")
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)